### PR TITLE
Serially start 5 containers

### LINKED ISSUE #2496: Smoke test from perf meeting
Serial start 5 long running containers, e.g. /bin/top or sleep, try a variety of images

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #2496

### CODE REVIEW

**Comment by hickeng** [comment]
It's not covered by the PR title but can we have parallel start of 5 containers as well?

**Comment by fdawg4l** [comment]
LGTM

**Comment by mhagen-vmware** [comment]
@hickeng done

File: tests/test-cases/Group1-Docker-Commands/1-5-Docker-Start.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ This test requires that a vSphere server is running and available
 6. Issue docker create vmware/photon
 7. Issue docker start vmware/photon <containerID>
 8. Issue docker start fakeContainer
+9. Create a container, remove it's ethernet adapter, then start the container
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-5-Docker-Start.md]: `its`

**Comment by anchal-agrawal** [comment]
Nitpick - can we have a blank line between test cases? That seems to be the case with other robot files.

LGTM

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**